### ISSUE TITLE
🐛 `rg --json | delta` output has different colour depending on filename ("ini")

### ISSUE THREAD

**Issue by tbm** [dandavison/delta#1513]
I don't know if this is a bug or intentional or something but it seems strange to me.  I used ripgrep on some files and noticed that one file had a different colour.  The file "ini" has different colours to any other files. (Maybe "init" is special cased?)

Reproduce:

```
$ echo "# [[liver]] {{gloss|organ of the body}}" > a
$ cp a ini
$ rg --json "\{" a | delta
$ rg --json "\{" ini | delta
```

![delta](https://github.com/dandavison/delta/assets/1556037/cb98c97a-6919-4685-90b5-81264a403aa0)

**Comment by tbm** [comment]
Original example where I noticed this:

![delta2](https://github.com/dandavison/delta/assets/1556037/e5abad83-b09e-4ef3-86b0-094c34aad57c)

It looks weird.

**Comment by th1000s** [comment]
Fixed with #1549, thank you for the report!
